pkg/secrets/filedriver/filedriver.go

```diff
@@ -33,6 +33,10 @@ type Driver struct {
 func NewDriver(rootPath string) (*Driver, error) {
 	fileDriver := new(Driver)
 	fileDriver.secretsDataFilePath = filepath.Join(rootPath, secretsDataFile)
+	if _, err := os.Stat(rootPath); os.IsNotExist(err) {
+		// the lockfile functions requre that the rootPath dir is executable
+		os.MkdirAll(rootPath, 0700)
+	}
```

Comment:
I suggest to just `os.MkdirAll(rootPath, 0o700)` without the `os.Stat`, because `os.MkdirAll` already does a stat:
https://golang.org/src/os/path.go?s=497:544#L18

Reply:
+1 to Sascha's suggestion.

We should also check the error as other code depends on it's success.

Reply:
Fixed, thanks!